widget/entry.go

```diff
@@ -161,10 +162,15 @@ func (e *Entry) Hide() {
 
 // SetText manually sets the text of the Entry to the given text value.
 func (e *Entry) SetText(text string) {
-	e.Text = text
+	runes := []rune(text)
+	e.updateRunes(runes)
+}
 
+// updateRunes updates the internal text to the given value
+func (e *Entry) updateRunes(runes []rune) {
```

Comment:
Please consider this name like a placeholder. Should we rename into something more self explaining? 
Example:
- `e.runes` -> `e.content`
- `updateRunes` -> `updateContent`

Reply:
renamed to updateText into 439ed1ff8a31a2eaf50ce7e43b8267043c63bb54